### ISSUE TITLE
the loss is unchanged

### ISSUE THREAD

**Issue by Rigel-1994** [ddtm/caffe#13]
It seems the code doesn't work. I follow the instrucment as the readme file but the loss is unchanged(dc_loss is almost 0.69+-, lp_loss is almost 3.36+-). What's wrong with that?

**Comment by congmonkey** [comment]
I have the same problem, do you solve it?

**Comment by eddielyc** [comment]
When training on office dataset, lp_loss will suddenly drop to 0.0+ after about 10000 iters. But the lp accuracy still stay low(about 0.6). In paper, the lp_loss should be 0.73 ??? I don't know whether I set hyperparameters correctly, all of settings are default.....
